Watchman.Discord/Areas/Initialization/Services/ServerScanningService.cs

```diff
@@ -22,30 +22,34 @@ public ServerScanningService(ICommandBus commandBus, MessagesHistoryService mess
 
         public async Task ScanChannelHistory(DiscordServerContext server, ChannelContext channel)
         {
-            const int LIMIT = 1000;
+            const int LIMIT = 10000;
```

Comment:
na niektórych serwerach tyle wiadomości to jest dziennie 

zrób jakiś milion czy coś tego typu

Reply:
```suggestion
            const int LIMIT = 100000;
```